sorting/tim_sort.c

```diff
@@ -0,0 +1,110 @@
+#include<stdio.h>
+#include<stdlib.h>
+
+const int RUN = 32; 
+
+int min(int a, int b)    
+{    
+    return a<b?a:b;     
+} 
+
+void insertionSort(int a[], int beg, int end) 
```

Comment:
Functions should be documented explaining the use of its parameters, a brief description, and what it returns.